test/gurka/test_build_landmark_database.cc

```diff
@@ -0,0 +1,61 @@
+#include <gtest/gtest.h>
```

Comment:
can we rename this file to `test_landmarks.cc` then we can put everything in here not just the builder